pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -55,28 +55,35 @@
     <RuntimeGroup Include="linuxmint.18">
       <Parent>ubuntu.16.04</Parent>
       <Architectures>x64</Architectures>
-      <Versions>1</Versions>
+      <Versions>1;2;3</Versions>
     </RuntimeGroup>
 
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4</Versions>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
 
     <!-- all opensuse releases are independent -->
     <RuntimeGroup Include="opensuse">
       <Parent>linux</Parent>
       <Architectures>x64</Architectures>
-      <Versions>13.2;42.1</Versions>
+      <Versions>13.2;42.1;42.2;42.3</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="sles">
```

Comment:
Make sure you are intentionally treating these as independent versions, eg: 12.2 shouldn't import 12.1.  If that shouldn't be true then remove `<TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>`.

Reply:
What should I be investigating to determine if they are "compatible" or not? It's not clear to me what that distinction is based on.

Reply:
This is related to my question below for Eric and Wes. Would be great if we could simply define sles 12. This is how the distro owners think of the major version 'band'. 12.2 is actually "12 SP2" and comes with the inherent compat promises of a service pack. FWIW SP3 is the latest SLES release.

Reply:
In the past we've left it up to the folks who know the most about the distros (ideally contributors to those).  For example: red hat goes out of there way to make their minor releases compatible across the major version (see https://access.redhat.com/articles/rhel-abi-compatibility), windows does so across all versions, whereas others like ubuntu does not and we've had a history of breaks when folks try to reuse binaries.

Ultimately its a bit of a loaded question: what is compatibility?  You can read through issues that mention `binary compatibility` to see some of the past discussion on the topic.  My preference is to opt for compatibility since it enables something that is otherwise not possible, see https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/readme.md#version-compatibility.

Reply:
It appears that SLES has a similar compatibility guarantees for releases within a major version: https://www.suse.com/partners/isv/porting_and_migration/

I'll remove the `<TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>` line.